source/agora/node/Ledger.d

```diff
@@ -99,6 +100,18 @@ public class Ledger
                 this.updateUTXOSet(block);
             }
         }
+
+        // restore validator set from lastest blocks
+        foreach (i; 0 .. ValidatorSet.ValidatorCycle)
+        {
+            if (i >= this.last_block.header.height)
```

Comment:
Shouldn't this be `>`?

If there are enrollment in the genesis block they will be skipped. Because the genesis block height is 0, and in the first loop `i` will be 0 so the `>=` condition is true and then it breaks.

Reply:
> Shouldn't this be >?

Yes, that should. I will fix the bug by changing the start value of the `foreach` into 1. Thanks.